pdns/lua-recursor4.cc

```diff
@@ -384,7 +384,10 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerMember("type", &DNSRecord::d_type);
   d_lw->registerMember("ttl", &DNSRecord::d_ttl);
   d_lw->registerMember("place", &DNSRecord::d_place);
-  
+
+  d_lw->registerMember("content", &EDNSOptionView::content);
```

Comment:
One easy way to resolve this would be to not register content with Luawrapper, but instead add a `get()` method to `EDNSOptionView` and register that one.

Then the Lua code could do this:

```lua
function gettag(remote, edns, loc, qname, qtype, ednsoptions)
    if ednsoptions ~= nil then 
        local opt = ednsoptions[65001] 
        if opt ~= nil and opt.length == 6 then
            print("Found option 65001", opt:get())
        end
    end 
    ...
end
```